src/main/scala/mesosphere/marathon/storage/repository/GcActor.scala

```diff
@@ -70,6 +72,7 @@ private[storage] class GcActor[K, C, S](
     val maxVersions: Int)(implicit val mat: Materializer, val ctx: ExecutionContext)
   extends FSM[State, Data] with LoggingFSM[State, Data] with ScanBehavior[K, C, S] with CompactBehavior[K, C, S] {
 
+  val ScanBatchSize = Try(context.system.settings.config.getInt("gc-actor-scan-batch-size")).toOption.getOrElse(32)
```

Comment:
why is this proper cased?  and not `scanBatchSize`?
and why is this not passed in the constructor like all the other values?
If passed via the constructor other values can be passed... and testing will be possible without need to setup details in the config object.